scrollreveal/scrollreveal.d.ts

```diff
@@ -0,0 +1,68 @@
+﻿// Type definitions for ScrollReveal
+// Project: https://github.com/jlmakes/scrollreveal.js
+// Definitions by: David Pires <https://github.com/Davidblkx>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace scrollReveal {
+    interface IScrollRevealRotateObject {
```

Comment:
https://www.typescriptlang.org/docs/handbook/writing-declaration-files.html#naming-conventions
> In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.